docs/guides/setup/configuring-metadata-stores/how-to-configure-a-validation-result-store-on-a-filesystem.md

```diff
@@ -2,4 +2,89 @@
 title: How to configure a Validation Result store on a filesystem
 ---
 
-This article is a stub.
+By default, Validation results are stored in the ``uncommitted/validations/`` directory.  Since Validations may include examples of data (which could be sensitive or regulated) they should not be committed to a source control system.  This guide will help you configure a new storage location for Validations on your filesystem.
+
+import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx';
+
+This guide will explain how to use a Validation Action to update Data Docs sites with new validation results from Validation Operator runs.
+
+<Prerequisites>
+
+- Configured a [Data Context](../../../tutorials/getting-started/initialize-a-data-context.md).
+ - Configured an [Expectation Suite ](../../../tutorials/getting-started/create-your-first-expectations).
+ - Configured a [Checkpoint](../../../guides/validation/checkpoints/how-to-create-a-new-checkpoint).
+ - Determined a new storage location where you would like to store Validations. This can either be a local path, or a path to a secure network filesystem.
+
+</Prerequisites>
+
+Steps
+-----
+
+1. **Configure a new folder on your filesystem where Validation results will be stored.**
+
+    Create a new folder where you would like to store your Validation results, and move your existing Validation results over to the new location. In our case, the name of the Validation result is ``npi_validations`` and the path to our new storage location is ``shared_validations/``.
+
+```bash
```

Comment:
What is our ultimate conclusion on indentation around code blocks? Should we just leave them as-is for now and revisit once we've determine a consistent style as a team?

Reply:
I tried to align them to look nicely with the text.  If they look nicely aligned, then we should leave them as is for now.

Reply:
That's a fair call. I tend to gravitate towards leftmost alignment just because Python snippets will work as intended but I think we can be flexible.

Please ignore this comment if desired.